src/bosh-dns/dns/server/upcheck.go

```diff
@@ -65,6 +67,29 @@ func (uc DNSAnswerValidatingUpcheck) IsUp() error {
 		return uc.wrapError(errors.New("DNS upcheck does not return the correct answer"))
 	}
 
+	request = dns.Msg{
+		Question: []dns.Question{
+			{Name: uc.internalDomain, Qtype: dns.TypeA},
```

Comment:
Some environments might be pure ipv6 (TypeAAAA). A TypeAny request would be one option.

Reply:
I don't think we have any Bosh deployments in the wild that are pure Ipv6, I don't think that would even work with bosh..